evaluators/rank/NdcgEvaluator/tests/test_ndcg_evaluator.py

```diff
@@ -8,15 +8,18 @@ def evaluator():
 
 @pytest.fixture(scope='function')
 def desired():
-    return [.8, .4, .1]
+    return [.8, .4, .1, 0]
 
-@pytest.mark.parametrize('actual, expected',[
-    ([.4, .1, .8], 0.795)
-    ([.0, .1, .4], 0.279)
-    ([.4, .1, .0], 0.396)
-])
+@pytest.mark.parametrize('actual, expected',
+    [
+        ([.4, .1, .8], 0.795),
+        ([.0, .1, .4], 0.279),
+        ([.4, .1, .0], 0.396),
+    ]
+)
 def test_encode_success(evaluator, actual, desired, expected):
```

Comment:
I am bit confused about the usage of 3 things actual, desired and expected. Can we have only actual score and expected score?

Reply:
expected here is the evaluation result expected, not the expected score